doc-content/enterprise-only/springboot/bus-app-security-disabling-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='bus-app-security-disabling-proc_{context}']
+
+= Disabling Spring Security in a {PRODUCT} business application
+You can configure Spring Security in a {PRODUCT} business application to provide the security context without authentication.
+//Why would I want to do this?
+
+.Prerequisites
+* You have a {PRODUCT} Spring Boot application.
+
+.Procedure
+. In the directory that contains your {PRODUCT} Spring Boot application, open the `business-application-service/src/main/java/com/company/service/DefaultWebSecurityConfig.java` file in a text editor or IDE.
+. Edit the `.antMatchers` method as shown in the following example:
++
+[source, java]
+----
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+
+                http
+                    .cors().and().csrf().disable()
+                    .authorizeRequests()
+                    .antMatchers("/*")
+                        .permitAll()
+                    .and().headers().frameOptions().disable();
+
+    }
+----
++
+The `PermitAll` method allows any and all requests for the specified URL pattern.
+
+NOTE: Because no security context is passed in the `HttpServletRequest`, Spring  creates an `AnonymousAuthenticationToken`
+and populates the `SecurityContext` with the `anonymousUser` user with no designated roles other than the `ROLE_ANONYMOUS` role. The user will not have access to many of the features of the application, for example they will be unable to action group assigned tasks.
```

Comment:
"they will be unable to action group assigned tasks." Seems like it should say something like "they will be unable to assign actions to group assigned tasks."